xspf2m3u.py

```diff
@@ -158,7 +166,14 @@ def main():
                 s = ' - '.join('{}: {}'.format(k, v) for k, v in track.items())
                 print('# Warning: ' + s)
             else:
-                print(location)
+                 if target == "M3U":
+                    print(location)
+                 if target == "M3UEXT":
+                    duration = int(int(track['duration']) / 1000) #I am not sure about this
+                    title = track['title']
+                    logo = track['image']
```

Comment:
There is no guarantee that all these fields are available.

- If the duration is unknown we can use -1
- If the title is unknown we can use the basename of `location`
- If the image is unknown we can skip it